backend/plugins/frontline_api/src/modules/inbox/trpc/inbox.ts

```
@@ -1,39 +1,1067 @@
+    conversationId: conversation._id,
+    userId,
+    customerId,
+    visitorId,
+    content,
+  });
+  return { conversation, message };
+};
+const t = initTRPC.context<ITRPCContext>().create();
 import { FrontlineTRPCContext } from '~/init-trpc';
 
 const t = initTRPC.context<FrontlineTRPCContext>().create();
```

Biome (1.9.4)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)